### PR TITLE
adding more language to security consideration

### LINKED ISSUE #36: Video requirements and encryption/decryption of metadata
https://datatracker.ietf.org/doc/html/draft-joras-sconepro-video-opt-requirements-00 - In this draft, they seem to talk about key exchange that we have kept out (implementation part)... But wonder if we have already incorporated the rest in our draft under "media" itself. Should we dig a little deeper into the cryptographic stuff or at least come up with a different draft for that?

### PR DESCRIPTION
**Pull request by sridharangirish91**

### CODE REVIEW

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md
Comment (boucadair) [inline draft-kwbdgrr-tsvwg-net-collab-rqmts.md]:
```suggestion
It is recommended to encrypt or obfuscate the metadata information so it is only available to hosts and to authorized network elements. The method of encryption or obfuscation is not described in this document but rather in other documents describing how this metadata is encoded and exchanged amongst hosts and network elements. The privacy implications of revealing metadata to network elements need to be thoroughly analyzed. This analysis should ensure that any exposure of metadata does not compromise user privacy or allow unauthorized entities to infer sensitive information about the data being transmitted while maintaining minimal resource consumption.
```

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-kwbdgrr-tsvwg-net-collab-rqmts.md
Comment (sridharangirish91) [reply draft-kwbdgrr-tsvwg-net-collab-rqmts.md]: Thanks Med!

**Review by tireddy2 — APPROVED** [review]

**Review by John-Kaippallimalil — APPROVED** [review]

**Comment by sridharangirish91** [comment]
@boucadair - I think this pr requires your approval to merge since your review is still pending. Please can you approve if everything looks good?

**Review by boucadair — APPROVED** [review]

**Pull request merged**
